flink-connector-jdbc/pom.xml

```diff
@@ -194,6 +194,22 @@ under the License.
 			<scope>test</scope>
 		</dependency>
 
+		<!-- clickhouse tests -->
+		<dependency>
+			<groupId>com.clickhouse</groupId>
+			<artifactId>clickhouse-jdbc</artifactId>
+			<version>0.4.6</version>
+			<scope>provided</scope>
+		</dependency>
+
+		<dependency>
+			<groupId>org.testcontainers</groupId>
+			<artifactId>clickhouse</artifactId>
+			<version>1.18.1</version>
```

Comment:
should be ${testcontainers.version}

Reply:
The testcontainer in the parent pom corresponds to ru.yandex.clickhouse.ClickHouseDriver, but this driver has stopped maintenance, so I have declared a new version of the image here, corresponding to the driver com.clickhouse.jdbc.ClickHouseDriver

Reply:
I hope you can give me some advice on how to handle it. Thank you very much

Reply:
we can update the version on parent

Reply:
If I modify this version number in parent pom.xml, it will affect other testcontainers

Reply:
the parent pom is 
```
<dependency>
                <groupId>org.testcontainers</groupId>
                <artifactId>testcontainers-bom</artifactId>
                <version>${testcontainers.version}</version>
                <type>pom</type>
                <scope>import</scope>
            </dependency>
```

Reply:
we have update master.. if you rebase will be available the last version o testcontainers

Reply:
ok,I have deleted the generated version